File: ServiceBook/ClassLibrary1/Models/SE.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ServiceBook.DAL.Models
{
    public class SE
    {
        [Key]
        public int SID { get; set; }
        public int EID { get; set; }
        [ForeignKey("SID")]
        public virtual Service Service { get; set; }
        [ForeignKey("EID")]
        public virtual Employee Employee { get; set; }
    }
}

File: ServiceBook/ServiceBook/Scripts/Service/Models/SE.js
function SE(data) {
    this.sid = null;
    this.eid = null;
    this.employee = null;
    this.service = null;
    if (data != null) {
        this.employee = new Employee(data.Employee);
        this.service = new Service(data.Service);
        this.sid = data.SID;
        this.eid = data.EID;
    }
}